pkgs/development/python-modules/azure-mgmt-media/default.nix

```diff
@@ -2,42 +2,42 @@
   lib,
   buildPythonPackage,
   fetchPypi,
+  setuptools,
   msrest,
-  msrestazure,
   azure-common,
   azure-mgmt-core,
-  pythonOlder,
+  typing-extensions,
 }:
 
 buildPythonPackage rec {
   pname = "azure-mgmt-media";
   version = "10.2.0";
-  format = "setuptools";
-
-  disabled = pythonOlder "3.6";
+  pyproject = true;
 
   src = fetchPypi {
     inherit pname version;
     extension = "zip";
     hash = "sha256-TVq/6dHttDGIUFzn8KTVeDTwcBMmphz3zrsGK7ux4aU=";
   };
 
-  propagatedBuildInputs = [
+  build-system = [ setuptools ];
+
+  dependencies = [
     msrest
-    msrestazure
     azure-common
     azure-mgmt-core
+    typing-extensions
   ];
 
   # has no tests
   doCheck = false;
```

Comment:
Non-actionable thing that I found interesting: There are tests, but they require network access to use. https://github.com/Azure/azure-sdk-for-python/tree/azure-mgmt-media_10.2.0/sdk/media/azure-mgmt-media/tests

Reply:
I'll update the comment. I'm pretty sure the comments are from some sort of template that was copied to make all of these, but it may not be accurate anymore after years of updates.